tests/org.eclipse.n4js.ui.tests/src/org/eclipse/n4js/tests/externalPackages/IndexSynchronizerPluginTest.java

```diff
@@ -67,8 +68,7 @@ public void testCleanRemovedNpm() throws Exception {
 
 		File prjDir = new File(getResourceUri(PROBANDS, SUBFOLDER));
 		IProject project = ProjectTestsUtils.importProject(prjDir, PROJECT_NAME);
-		IResourcesSetupUtil.fullBuild();
-		waitForAutoBuild();
+		testedWorkspace.fullBuild();
```

Comment:
So, what's the conclusion for writing tests in the future?

Can the guideline be summarized as: "use JUnit rule `TestedN4JSWorkspace` whenever possible!" ?
And this applies to all plugin and pluginUI that build something, right?

Reply:
Yes, that would be a valid guideline.